R/agglomerate.R

```diff
@@ -45,26 +45,26 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting
 #'        whether to remove those columns of rowData that include only NAs after
-#'        agglomeration. (By default: \code{remove_empty_ranks = FALSE})
-#'        \item \code{make_unique}: A single boolean value for selecting
-#'        whether to make rownames unique. (By default: \code{make_unique = TRUE})
+#'        agglomeration. (By default: \code{empty.ranks.rm = FALSE})
+#'        \item \code{make.unique}: A single boolean value for selecting
```

Comment:
Instead of `make.unique` we could just use `unique`? I think it is used with that meaning also in some other packages.

Reply:
On the other hand the purpose and name matches with make.unique() function https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/make.unique

Reply:
ok